cmd/kubelet/app/options/options.go

```diff
@@ -535,6 +535,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 
 	fs.StringVar(&c.ResolverConfig, "resolv-conf", c.ResolverConfig, "Resolver configuration file used as the basis for the container DNS resolution configuration.")
 	fs.BoolVar(&c.CPUCFSQuota, "cpu-cfs-quota", c.CPUCFSQuota, "Enable CPU CFS quota enforcement for containers that specify CPU limits")
+	fs.DurationVar(&c.CPUCFSQuotaPeriod.Duration, "cpu-cfs-quota-period", c.CPUCFSQuotaPeriod.Duration, "Sets CPU CFS quota period value, cpu.cfs_period_us, defaults to Linux Kernel default")
```

Comment:
`defaults to Linux Kernel default (100ms)` for clear?

Reply:
See the former comment from ligitt. Cobra will render the default automatically.

Reply:
We should stop adding new flags to kubelet. Add directly to `KubeletConfiguration` instead?

Reply:
i would prefer to not have the flag and just use via `KubeletConfiguration`; we should note that dynamically changing the field value will not impact currently running pods (only new pods)